back/Gemfile

```diff
@@ -94,7 +94,10 @@ gem 'activerecord_json_validator', '~> 1.3.0'
 gem 'activerecord-postgis-adapter', '~> 7.0.0'
 
 # Processing images
-gem 'carrierwave', '~> 2.2'
+# We need `config.download_retry_count`.
+# It should be available in the version next to 2.2.2
```

Comment:
Thank you for leaving a note why we're pinning to a specific version here, appreciate it!